std_msgs/Float64MultiArray cmd
---
std_msgs/Bool success